crates/bevy_ecs/src/query/state.rs

```diff
@@ -909,7 +908,7 @@ impl<Q: WorldQuery, F: WorldQuery> QueryState<Q, F> {
         change_tick: u32,
     ) {
         // NOTE: If you are changing query iteration code, remember to update the following places, where relevant:
-        // QueryIter, QueryIterationCursor, QueryState::for_each_unchecked_manual, QueryState::many_for_each_unchecked_manual, QueryState::par_for_each_unchecked_manual
```

Comment:
instead of removing this I think it needs to be updated to talk about `QueryManyIter::fetch_next_aliased_unchecked`